DSA-Campaign-Uplift-Estimation/src/main/java/com/google/sps/servlets/DSACampaign.java

```diff
@@ -0,0 +1,52 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+// contains all the necessary information about a DSA campaign, including id's, input settings, and statistics
+public class DSACampaign {
+
+	public int DSACampaignId;
+    public int userId;
+    public int keywordCampaignId;
+
+    // settings
```

Comment:
Would be good to have some place where each setting is explained (maybe here).

Reply:
All of the settings are temporary right now - we still need to formalize exactly all the settings we are going to use. Right now we are just trying to establish minimum functionality - I will create a TODO however to explain the settings once we finally decide on them.

Reply:
I think it would be best to formalize the settings first before moving on with coding. Let us discuss in the pod sync.